cdb2jdbc/src/main/java/com/bloomberg/comdb2/jdbc/Comdb2Handle.java

```diff
@@ -164,6 +165,7 @@ public Comdb2Handle duplicate() {
     public Comdb2Handle(String dbname, String cluster) {
         super(new ProtobufProtocol(), null);
         sets = new ArrayList<String>();
```

Comment:
JDBC requires statement-level effects, not transaction-level effects.

Reply:
The JDBC driver forces the database into statement-level query effects via the new "set queryeffects statement" directive.  A user can force it back into the old behavior by issuing a "set queryeffects transaction"